runtime/basilisk/src/lib.rs

```diff
@@ -106,7 +106,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("basilisk"),
 	impl_name: create_runtime_str!("basilisk"),
 	authoring_version: 1,
-	spec_version: 26,
+	spec_version: 27,
```

Comment:
We don't need to bump this since there is no change to runtime. Is there?

Reply:
not really, this was changed for testing purposes